docs/tables/mongodbatlas_advanced_cluster.md

```diff
@@ -0,0 +1,16 @@
+# Table: mongodbatlas_advanced_cluster
+
+MongoDB Atlas Cluster is a NoSQL Database-as-a-Service offering in the public cloud (available in Microsoft Azure, Google Cloud Platform, Amazon Web Services).
+
+## Examples:
+
```

Comment:
We're missing the `### Basic info` example, which is mentioned in https://steampipe.io/docs/develop/table-docs-standards#basic-info-example